spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.spec.js

```diff
@@ -4,7 +4,6 @@ describe('Google Tag Manager page view tracking', function () {
   var GOVUK = window.GOVUK
   var saved = {}
   var expected
-  var nullValue = null
```

Comment:
Could this not be changed to `undefined`? I assume there was some kind of weirdness involved?

Reply:
@andysellick Yep, `standardx` said `ga4-page-views.spec.js:7:7: It's not necessary to initialize 'nullValue' to undefined.`

Reply:
Interesting... wonder why it also didn't complain about setting `this.nullValue` in the page view tracker? Ah well, fair enough.